scheduler_failover_controller/failover/failover_controller.py

```diff
@@ -160,35 +193,75 @@ def is_scheduler_running(self, host):
             # todo: If there's more then one scheduler running this should kill off the other schedulers. MIGHT ALREADY BE HANDLED. DOUBLE CHECK.
 
             is_running = active_list_length > 0
+            if (is_running):
+                url = "http://" + \
```

Comment:
Can we confirm what version of Airflow this started coming with so we know what version of Airflow the ASFOC will work with?

Reply:
The /health endpoint existed before but the info about the scheduler in the output was added in 1.10.2.

[https://issues.apache.org/jira/browse/AIRFLOW-3504](https://issues.apache.org/jira/browse/AIRFLOW-3504)